nethack.yaml

```
@@ -0,0 +1,86 @@
+package:
+  name: nethack
+  version: 3.6.7
+  epoch: 0
+  description: The dungeon exploration game
+  copyright:
+    - license: NGPL
+
+environment:
+  contents:
+    packages:
+      - bison
+      - build-base
+      - flex
+      - ncurses-dev
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NetHack/NetHack
+      tag: NetHack-${{package.version}}_Released
+      expected-commit: ed600d9f0f3c37677418f0150f59363ca641f3dc
+
+  # Based on https://gitlab.alpinelinux.org/alpine/aports/-/blob/cc26e248bce55c5f6a475f40ad8c16c5a1a3df58/community/nethack/APKBUILD
+  - runs: |
+      # Fix linking against debian-specific tinfo
+      sed -e 's|^WINTTYLIB.*|WINTTYLIB = -lncurses|' -i sys/unix/hints/linux
+      # Fix nonstandard cp usage
+      sed -e 's/cp -n/cp/g' -i sys/unix/hints/linux
+      # Configuration stuff
+      sed -e 's|^/\* \(#define LINUX\) \*/|\1|' \
+        -i include/unixconf.h
+      # we are setting up for setgid games, so modify all necessary permissions
+      # to allow full access for groups
+      sed -e '/^HACKDIR/ s|/games/lib/\$(GAME)dir|/var/games/nethack/|' \
+        -e '/^SHELLDIR/ s|/games|/usr/bin|' \
+        -e '/^VARDIRPERM/ s|0755|0775|' \
+        -e '/^VARFILEPERM/ s|0600|0664|' \
+        -e '/^GAMEPERM/ s|0755|02755|' \
+        -e 's|\(DSYSCF_FILE=\)\\"[^"]*\\"|\1\\"/var/games/nethack/sysconf\\"|' \
+        -e 's|\(DHACKDIR=\)\\"[^"]*\\"|\1\\"/var/games/nethack/\\"|' \
+        -i sys/unix/hints/linux
+      sed -e 's|^#GAMEUID.*|GAMEUID = root|' \
+        -e 's|^#GAMEGRP.*|GAMEGRP = games|' \
+        -e '/^FILEPERM\s*=/ s|0644|0664|' \
+        -e '/^DIRPERM\s*=/ s|0755|0775|' \
+        -i sys/unix/Makefile.top
+      sed -e "/^MANDIR\s*=/s|/usr/man/man6|${{targets.destdir}}/usr/share/man/man6|" \
+        -i sys/unix/Makefile.doc
+
+      cd ./sys/unix
+      sh setup.sh hints/linux
+      cd ../..
+      make
+
+      install -dm755 ${{targets.destdir}}/usr/share/man/man6
+      install -dm755 ${{targets.destdir}}/usr/share/doc/nethack
```

Can this be moved to.a separate sub-packages under nethack-doc
```suggestion
      install -dm755 ${{targets.destdir}}/usr/share/man/man6
      install -dm755 ${{targets.destdir}}/usr/share/doc/nethack
```

---

Good call; done!